components/scripts/lib/build-scan-online.sh

```diff
@@ -24,10 +24,14 @@ read_build_scan_metadata() {
       debug ""
     fi
 
-    while IFS=, read -r run_num field_1 field_2 field_3; do
-       base_urls[$run_num]="$field_1"
-       build_scan_urls[$run_num]="$field_2"
-       build_scan_ids[$run_num]="$field_3"
+    local run_num project_name base_url build_scan_url build_scan_id
```

Comment:
@jthurne Is this sort of what you had in mind for updating the Build Scan parsing logic? Since I was modifying the `build-scans.csv` code I figured I could at least clean this up a little bit.

Reply:
I'm not Jim :-) but it reads nicely since one can now understand what the expected fields are.

Reply:
I'll close this thread.